remake_wqp.yml

```diff
@@ -48,3 +50,10 @@ targets:
     
   secchi.clean:
     command: munge_wqp(secchi.merged)
+    depends: 
+      - scripts/download_munge_wqp.R
```

Comment:
I don't think you need the explicit depends here as your sources are up above, do you? I think if the function changes, it's re-run.